### PR TITLE
Add 'Institute of Chartered Accountants of Scotland' to institutions

### PR DESCRIPTION
**Pull request by jhenderson**
### Context
https://trello.com/c/9qNctNxM/4652-register-trainee-teachers

A provider cannot find the closest match, so a request has been made by support to add it to the list of institutions.

### Changes proposed in this pull request
- Add `The Institute of Chartered Accountants of Scotland` to `INSTITUTIONS`

### CODE REVIEW

**Comment by frankieroberto** [comment]
@jhenderson can you confirm whether this institution has degree awarding powers in Scotland or not? They don't seem to be in this list, but possibly that's not up-to-date? https://www.gov.scot/policies/universities/

If they don't award "degrees" but do award equivalent qualifications, we can still add them to the list, adding `'has_never_awarded_degrees' => true`. This helps us validate degree qualifications (in theory).

**Comment by edwardhorsford** [comment]
I don't think we should add this. This list aims to have all UK degree awarding institutions - which I don't think this is.

This has come about because a provider has attempted to add an accounting qualification to Register. But I suspect we don't want to know about the accounting qualification as it isn't relevant to their teacher training.

What this does highlight is that Register (and this list) don't have any generic options for where an institution is missing - we should add something for this.

**Comment by edwardhorsford** [comment]
To summarise some offline conversations:

This institution isn't degree awarding, so arguably doesn't belong in this list. However, the list already contains many non-degree awarding institutions as a result of HESA including them - so one more doesn't make a big difference. We have a flag that can mark something as non-degree awarding. As this is blocking a user from proceeding, we'll add the item. Services can choose to suppress non-degree awarding institutions from appearing if they choose.

In short term we will also look at adding a 'catch all' generic option of "Other institution" - which could be used in cases like this.

**Review by alaric-dfe — COMMENTED** [review]
Thanks for fixing the hash keys! Not sure how that slipped through in the original data import... Looks good to me.
